tracing-subscriber/src/fmt/time/mod.rs

```diff
@@ -15,6 +16,19 @@ pub use time_crate::UtcTime;
 #[cfg_attr(docsrs, doc(cfg(all(unsound_local_offset, feature = "local-time"))))]
 pub use time_crate::LocalTime;
 
+/// [`chrono`]-based implementation for [`FormatTime`].
+#[cfg(feature = "chrono")]
+#[cfg_attr(docsrs, doc(cfg(feature = "local-time")))]
```

Comment:
this is a private module, so it doesn't need a `doc(cfg(...))` attribute. not a big deal.

also, it seems odd that this has a `doc(cfg(feature = "local-time"))` when this module does not actually require that feature.

Reply:
that was me copy-and-pasting incorrectly. sorry.